semgrep/tests/e2e/snapshots/test_check/test_max_memory/error.txt

```diff
@@ -36,10 +36,10 @@ Files skipped:
    • <none>
 
 
-skipped: all .gitignored files
-for a detailed list of skipped files, run semgrep with the --verbose flag
+Some files were skipped.
+  Scan was limited to files tracked by git.
 
-ran 1 rules on 1 files: 0 findings
+Ran 1 rule on 1 file: 0 findings.
```

Comment:
Why the period? :P

Reply:
Most other lines are full sentences & I like the style consistency.